### DISCUSSION TITLE
body template property of an array.

### DISCUSSION THREAD

**Discussion by ibmoreno** [mockoon/mockoon#908] [Help]
hi, how to get the body template property of an array. example [{"firstname":"ivan","lastname":"test"}]?

reponse:
{
 "nome": "{{body 'firstname' }}",
}

**Comment by 255kb** [comment]
Hi, you need to target the array item id: `{{body '0.firstname'}}`.

**Reply by ibmoreno** [reply]
I've tried that and it doesn't work, it returns empty ""

**Reply by ibmoreno** [reply]
ignore, it worked, I was sending it via postmaim and I had forgotten to set it as application/json. forgive me.. thank you very much

**Comment by 255kb** [comment]
Glad you found the reason ☺️
